http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/dao/google/MockGoogleDiskService.scala

```diff
@@ -0,0 +1,29 @@
+package org.broadinstitute.dsde.workbench.leonardo.dao.google
+
+import cats.effect.IO
+import cats.mtl.ApplicativeAsk
+import com.google.cloud.compute.v1.{Disk, Operation}
+import fs2.Stream
+import org.broadinstitute.dsde.workbench.google2.{DiskName, GoogleDiskService, ZoneName}
+import org.broadinstitute.dsde.workbench.model.TraceId
+import org.broadinstitute.dsde.workbench.model.google.GoogleProject
+
+class MockGoogleDiskService extends GoogleDiskService[IO] {
```

Comment:
we should probably have added this in wb-libs similar to https://github.com/broadinstitute/workbench-libs/blob/develop/google2/src/test/scala/org/broadinstitute/dsde/workbench/google2/mock/FakeGoogleStorageInterpreter.scala

Reply:
I can add a `TODO` similar to https://github.com/DataBiosphere/leonardo/blob/develop/http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/dao/google/MockGoogleComputeService.scala#L24 and we can create a tech debt ticket to move them both to wb-libs